pkgs/development/python-modules/opensfm/0005-fix-numpy-2-test-failures.patch

```diff
@@ -0,0 +1,239 @@
+diff --git a/opensfm/exif.py b/opensfm/exif.py
+--- a/opensfm/exif.py
++++ b/opensfm/exif.py
+@@ -509,7 +509,7 @@ class EXIF:
+                     )
+                 )
+ 
+-            if np.all(ypr) is not None:
++            if np.all(ypr != None):
```

Comment:
God this is cursed. So this is `dtype("O")`? What is the status with upstreaming this?